Week3/Homework/Exercise-5-Total-cost-is.js

```diff
@@ -0,0 +1,25 @@
+function calculateTotalPrice(order) {
+
+    let totalPrice = 0;
+    for (let i in order) {
+
+        totalPrice = totalPrice + order[i];
+    }
+
+    return "The total price of all items is : " + totalPrice + " $."
```

Comment:
Some unnecessary spaces towards the beginning and end of the function. But 👍 for logic. Also, I would suggest you had a look into `Array.prototype.reduce` and `Object.prototype.keys` methods to achieve a similar behaviour. 🙂